build.sbt

```diff
@@ -4,11 +4,11 @@ version := "0.1"
 
 organization := "edu.berkeley.cs.amplab"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.10"
 
 spName := "amplab/opaque"
 
-sparkVersion := "2.4.0"
+sparkVersion := "3.0.0"
```

Comment:
Perhaps we could move the Spark 3.0-related changes to a different PR. That way we can differentiate bugs arising from each one and even revert each one individually if necessary.

Reply:
Looks like this is still unaddressed?

Reply:
Moved to #94.